### ISSUE TITLE
Crash when rendering 2022.20 - 2022.22

### ISSUE THREAD

**Issue by TechplexEngineer** [smeighan/xLights#3558]
I'm seeing the same issue as @cybie257 as quoted below. 

        I can confirm that loading a file in the Linux version 2022.20 and above that loading a file, clicking on an element (which would begin animation), would freeze and force a force quit. This is with the AppImage and Compiled version(s). Something changed in 2022.20 as 2022.19 has no issues noted in this thread. I have found 2022.20, 2022.21, and 2022.22 all have this symptom. 

Per Bill comment, I will try the nightly build option, but that would mean 2022.22 should be where a fix is implemented and still has the issue.

EDIT: I wasn't finding this to occur on the Windows side of things (as of yet)... 
EDIT 2: Tried nightly build. Same issues persist. 
FINAL EDIT: Since OPs Issue is Windows Specific, this appears to be a separate issue I can't duplicate on Windows with 2022.20/21/22 I will open a new Issue when I gather logs and more information during trouble shooting. For now, seems something definitely broke after 2022.19 with the Picture Element when rendering.

_Originally posted by @cybie257 in https://github.com/smeighan/xLights/issues/3513#issuecomment-1308983004_

Relevant Log Section
```
2022-11-13 20:01:02,558 140088844624128 log_base [DEBUG] Rendering all.
2022-11-13 20:01:02,558 140088844624128 log_base [DEBUG] Model blending: On
2022-11-13 20:01:02,558 140088844624128 log_base [DEBUG] xLightsFrame::RenderIseqData bottom_layers 1
2022-11-13 20:01:02,558 140088844624128 log_base [DEBUG] xLightsFrame::RenderIseqData clearing sequence data.
2022-11-13 20:01:02,565 140088844624128 log_base [INFO]    iseq below effects done.
2022-11-13 20:01:02,566 140088844624128 log_base [DEBUG] Render tree built. 23 entries.
2022-11-13 20:01:02,566 140088844624128 log_base [DEBUG] Rendering 23 models 4552 frames.
2022-11-13 20:01:02,573 140088844624128 log_base [WARN] Model::InitRenderBufferNodes firstNode + Nodes.size() = 0. ModelGroup::'Flood Lights'. This commonly happens on a polyline segment with zero pixels or a custom model with no nodes but with effects dropped on it.
2022-11-13 20:01:02,573 140088844624128 log_base [WARN] Model::InitRenderBufferNodes firstNode + Nodes.size() = 0. ModelGroup::'Verticals'. This commonly happens on a polyline segment with zero pixels or a custom model with no nodes but with effects dropped on it.
2022-11-13 20:01:02,573 140088844624128 log_base [WARN] Model::InitRenderBufferNodes firstNode + Nodes.size() = 0. ModelGroup::'Verticals'. This commonly happens on a polyline segment with zero pixels or a custom model with no nodes but with effects dropped on it.
2022-11-13 20:01:02,573 140088844624128 log_base [WARN] Model::InitRenderBufferNodes firstNode + Nodes.size() = 0. ModelGroup::'Verticals'. This commonly happens on a polyline segment with zero pixels or a custom model with no nodes but with effects dropped on it.
2022-11-13 20:01:02,573 140088844624128 log_base [WARN] Model::InitRenderBufferNodes firstNode + Nodes.size() = 0. ModelGroup::'Horizontals'. This commonly happens on a polyline segment with zero pixels or a custom model with no nodes but with effects dropped on it.
2022-11-13 20:01:02,573 140088844624128 log_base [WARN] Model::InitRenderBufferNodes firstNode + Nodes.size() = 0. ModelGroup::'Horizontals'. This commonly happens on a polyline segment with zero pixels or a custom model with no nodes but with effects dropped on it.
2022-11-13 20:01:02,573 140088844624128 log_base [WARN] Model::InitRenderBufferNodes firstNode + Nodes.size() = 0. ModelGroup::'Horizontals'. This commonly happens on a polyline segment with zero pixels or a custom model with no nodes but with effects dropped on it.
2022-11-13 20:01:02,573 140088844624128 log_base [WARN] Model::InitRenderBufferNodes firstNode + Nodes.size() = 0. ModelGroup::'Horizontals'. This commonly happens on a polyline segment with zero pixels or a custom model with no nodes but with effects dropped on it.
2022-11-13 20:01:02,573 140088844624128 log_base [WARN] Model::InitRenderBufferNodes firstNode + Nodes.size() = 0. ModelGroup::'Horizontals'. This commonly happens on a polyline segment with zero pixels or a custom model with no nodes but with effects dropped on it.
2022-11-13 20:01:02,577 140086884669184 log_base [CRIT] Crashed: Fatal exception occurred.
2022-11-13 20:01:02,579 140088844624128 log_base [DEBUG] Job pool new size 15.
2022-11-13 20:01:02,621 140086884669184 log_base [CRIT] xLights version 2022.22 64bit
Time: 2022-11-13T20:01:02
[xLights_l4cpp_2.log](https://github.com/smeighan/xLights/files/9998732/xLights_l4cpp_2.log)

[xLights_l4cpp_1.log](https://github.com/smeighan/xLights/files/9998730/xLights_l4cpp_1.log)

<email>removed</email>
Crashed Thread ID: 0x7f6884ff9700
```
[xLights_l4cpp.log](https://github.com/smeighan/xLights/files/9998729/xLights_l4cpp.log)

Show directory: https://github.com/TechplexEngineer/Lightshow/tree/main/sequences

**Comment by iXware71** [comment]
the same issue i get on Linux (opensuse leap 15.4) appimage  -  the model is a tree at WLED (ESP32) with 1 string, 8 strands and 400 LED - if this is interesting

2022-11-14 10:36:25,645 140670552363264 log_base [DEBUG] Rendering all.
2022-11-14 10:36:25,645 140670552363264 log_base [DEBUG] Model blending: On
2022-11-14 10:36:25,645 140670552363264 log_base [DEBUG] xLightsFrame::RenderIseqData bottom_layers 1
2022-11-14 10:36:25,645 140670552363264 log_base [DEBUG] xLightsFrame::RenderIseqData clearing sequence data.
2022-11-14 10:36:25,645 140670552363264 log_base [INFO]    iseq below effects done.
2022-11-14 10:36:25,646 140670552363264 log_base [DEBUG] Render tree built. 1 entries.
2022-11-14 10:36:25,646 140670552363264 log_base [DEBUG] Rendering 1 models 1200 frames.
2022-11-14 10:36:25,647 140669775509248 log_base [CRIT] Crashed: Fatal exception occurred.
2022-11-14 10:36:25,648 140670552363264 log_base [DEBUG] Job pool new size 4.
2022-11-14 10:36:25,701 140669775509248 log_base [CRIT] xLights version 2022.22 64bit
Time: 2022-11-14T10:36:25
Crashed Thread ID: 0x7ff03bfff700

/tmp/.mount_xLightWK6xgo/AppRun.wrapped(_ZN14xlCrashHandler11HandleCrashEbRKNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEE+0x1009) [0x562fc4502729]
/tmp/.mount_xLightWK6xgo/AppRun.wrapped(_ZN11xlGLBaseApp16OnFatalExceptionEv+0x52) [0x562fc3a86e82]
/tmp/.mount_xLightWK6xgo/usr/bin/../lib/libwx_gtk3u-3.3.so.0(+0x6fee26) [0x7ff0781bae26]
/lib64/libpthread.so.0(+0x168c0) [0x7ff07927d8c0]
/tmp/.mount_xLightWK6xgo/AppRun.wrapped(_ZN9RenderJob11GetwxStatusEv+0x51) [0x562fc3ea0cb1]
/tmp/.mount_xLightWK6xgo/AppRun.wrapped(_ZN9RenderJob11LogToLoggerEi+0xb3) [0x562fc3ea2023]
/tmp/.mount_xLightWK6xgo/AppRun.wrapped(_ZN9RenderJob7ProcessEv+0xc4) [0x562fc3ea5514]
/tmp/.mount_xLightWK6xgo/AppRun.wrapped(_ZN13JobPoolWorker10ProcessJobEP3Job+0x24d) [0x562fc3fea66d]
/tmp/.mount_xLightWK6xgo/AppRun.wrapped(_ZN13JobPoolWorker5EntryEv+0xf7) [0x562fc3feaa77]
/tmp/.mount_xLightWK6xgo/AppRun.wrapped(+0x1427c8a) [0x562fc3feac8a]
/usr/lib64/libstdc++.so.6(+0xdb3d4) [0x7ff0774833d4]
/lib64/libpthread.so.0(+0xa6ea) [0x7ff0792716ea]
/lib64/libc.so.6(clone+0x3f) [0x7ff076b66a6f]

**Comment by billnapier** [comment]
I'm also seeing this on 2022.23.

```
2022-11-17 22:03:15,733 140443060707584 log_base [DEBUG] Render tree built for model MegaTree 950ms-1950ms. 3 entries.
2022-11-17 22:03:15,733 140443060707584 log_base [DEBUG] Rendering 1 models 21 frames.
2022-11-17 22:03:15,734 140441924130560 log_base [CRIT] Crashed: Fatal exception occurred.
2022-11-17 22:03:15,734 140443060707584 log_base [DEBUG] Job pool new size 4.
2022-11-17 22:03:15,736 140443060707584 log_opengl [DEBUG] Depth of 8 supported, using it
2022-11-17 22:03:15,736 140443060707584 log_base [DEBUG]                     Creating GL Canvas for EmptyGrid
2022-11-17 22:03:15,841 140441924130560 log_base [CRIT] xLights version 2022.23 64bit
Time: 2022-11-17T22:03:15
Crashed Thread ID: 0x7fbb2effd700

/tmp/.mount_xLight6OwTvo/AppRun.wrapped(_ZN14xlCrashHandler11HandleCrashEbRKNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEE+0x1009) [0x55605271c729]
/tmp/.mount_xLight6OwTvo/AppRun.wrapped(_ZN11xlGLBaseApp16OnFatalExceptionEv+0x52) [0x556051c9ebc2]
/tmp/.mount_xLight6OwTvo/usr/bin/../lib/libwx_gtk3u-3.3.so.0(+0x6fee26) [0x7fbb7ce84e26]
/lib/x86_64-linux-gnu/libpthread.so.0(+0x153c0) [0x7fbb7df6a3c0]
/tmp/.mount_xLight6OwTvo/AppRun.wrapped(_ZN9RenderJob11GetwxStatusEv+0x51) [0x5560520b9921]
/tmp/.mount_xLight6OwTvo/AppRun.wrapped(_ZN9RenderJob11LogToLoggerEi+0xb3) [0x5560520bac93]
/tmp/.mount_xLight6OwTvo/AppRun.wrapped(_ZN9RenderJob7ProcessEv+0xc4) [0x5560520be184]
/tmp/.mount_xLight6OwTvo/AppRun.wrapped(_ZN13JobPoolWorker10ProcessJobEP3Job+0x24d) [0x5560522039dd]
/tmp/.mount_xLight6OwTvo/AppRun.wrapped(_ZN13JobPoolWorker5EntryEv+0xf7) [0x556052203de7]
/tmp/.mount_xLight6OwTvo/AppRun.wrapped(+0x1429ffa) [0x556052203ffa]
/usr/lib/x86_64-linux-gnu/libstdc++.so.6(+0xd6de4) [0x7fbb7c574de4]
/lib/x86_64-linux-gnu/libpthread.so.0(+0x9609) [0x7fbb7df5e609]
/lib/x86_64-linux-gnu/libc.so.6(clone+0x43) [0x7fbb7c264293]
```

**Comment by TechplexEngineer** [comment]
[xLights_dbgrpt-995145-20221130T205757.zip](https://github.com/smeighan/xLights/files/10128672/xLights_dbgrpt-995145-20221130T205757.zip)

**Comment by TechplexEngineer** [comment]
This seems to be fixed in 2022.24
